proto/backend.proto

```diff
@@ -110,14 +110,26 @@ message QueryDataRequest {
 }
 
 message QueryDataResponse {
-  // Arrow encoded DataFrames
-  // Each frame encodes its own: Errors, meta, and refId
-  repeated bytes frames = 1;
+  // List of data query responses
+  repeated DataResponse responses = 1;
```

Comment:
should this be a list?  or a Map<refId,DataResponse>?  a list allows for the case where refIds overlap so list lis likely better

Reply:
That Map version would fit Grafana server's would map closer to Grafana server's current tsdb.Response. Would make the change in Grafana server a lot easier I think.